Solutions/Network Session Essentials/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "publisherId": "azuresentinel",
+  "offerId": "azure-sentinel-solution-networksession-preview",
```

Comment:
@devikamehra, please remove -preview, thanks.

Reply:
The solution is in preview right now. Should this be removed?